src/i18n/en.json

```
@@ -1204,6 +1220,11 @@
       "shrink": "Shrink qualities",
       "comment": "Comments"
     },
+    "cut": {
+      "form": {
+        "value": "Value"
```

```suggestion
        "value": "Nombre"
```
Désolé je ne sais pas ce que je veux :/

---

🤣 t'inquiète c'est rapide